README.rst

```diff
@@ -317,16 +316,16 @@ Install Kokkos (Optional)
 We suggest first installing Kokkos with the wanted configuration following the instructions found in the `Kokkos documentation <https://kokkos.github.io/kokkos-core-wiki/building.html>`_.
 For example, the following will build Kokkos for NVIDIA A100 cards
 
-Download the `Kokkos code <https://github.com/kokkos/kokkos/releases>`_. Lightning Kokkos was tested with Kokkos version <= 4.5.0
+Download the `Kokkos code <https://github.com/kokkos/kokkos/releases>`_. Lightning-Kokkos was tested with Kokkos version <= 4.5.0
 
 .. code-block:: bash
 
     # Replace x, y, and z by the correct version
-    wget https://github.com/kokkos/kokkos/archive/refs/tags/4.x.yz.tar.gz
+    wget https://github.com/kokkos/kokkos/archive/refs/tags/4.x.y.z.tar.gz
```

Comment:
```suggestion
    wget https://github.com/kokkos/kokkos/archive/refs/tags/4.x.yz.tar.gz
```

Reply:
may need corresponding changes for the context below